packages/at_persistence_spec/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.0.0
+- feat!: major version change for simplified keystore interface
```

Comment:
What's the ! for?

Reply:
> What's the ! for?

@cpswan recommendation from @XavierChanth 
> I would like to adopt the notation of using ! to represent breaking changes, I've seen many semantic commit projects using this. feat: is non-breaking, feat!: is breaking.